src/workflows/pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "airqo_etl_utils"
+version = "1.0.0"
+description = "AirQo Airflow custom python package"
+authors = ["AirQo <data@airqo.net>"]
+license = "MIT"
+keywords = ["python", "airflow", "AirQo"]
+include = ["**/*.json"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+apache-airflow = "*"
+apache-airflow-providers-slack = "*"
+confluent_avro = "*"
+google-cloud-bigquery = "*"
```

Comment:
google-cloud-bigquery = ">=3.0.0,<4.0.0"

Reply:
On it, Thanks for the feedback